assets/javascripts/kitten/components/simulators/loan-simulator.js

```diff
@@ -103,26 +110,61 @@ class LoanSimulator extends React.Component {
       return this.props.amountOutOfBoundsError
   }
 
-  installmentMin() {
-    const installmentStep = this.installmentStep()
-    const value = this.state.amount / this.props.durationMax
-    const min = Math.ceil(value / installmentStep) * installmentStep
-    if (min > this.state.amount * 1)
-      return this.state.amount
-    else
-      return min
+  render() {
+    return (
+      <LoanSimulatorContent
+        { ...this.props }
+        { ...this.state }
+        handleFocus={ this.handleFocus }
+        handleAmountChange={ this.handleAmountChange }
+        handleAmountKeyDown={ this.handleAmountKeyDown }
+        handleInstallmentLabelClick={ this.handleInstallmentLabelClick }
+        handleInstallmentChange={ this.handleInstallmentChange }
+        handleInstallmentAction={ this.handleInstallmentAction }
+        duration={ this.duration() }
+        commissionAmount={ this.commissionAmount() }
+        commissionRate={ this.commissionRate() }
+        installmentMin={ this.installmentMin() }
+        installmentMax={ this.installmentMax() }
+        installmentStep={ this.installmentStep() }
+        error={ this.error() }
+        amountError={ this.amountError() } />
+    )
   }
+}
 
-  installmentMax() {
-    return this.state.amount * 1
+class LoanSimulatorContent extends React.Component {
```

Comment:
Question nomenclature, on choisit d'avoir : 
- `LoanSimulator` et `LoanSimulatorContent` 
- ou `LoanSimulatorContainer` et `LoanSimulator`

Sachant qu'un composant présentationnel peut contenir des composants container (cf. https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.cspn02v20).